src/Custom-sample.h

```diff
@@ -209,10 +209,11 @@
    CSS / template
  #######################################################################################################
  */
+
+#define WEBPAGE_TEMPLATE_DEFAULT_HEADER "<header class='headermenu'><h1>Kretz IoT: {{title}}</h1><BR>"
+#define WEBPAGE_TEMPLATE_DEFAULT_FOOTER "<footer><br><h6>Server: <a href='https://openhab:8443' style='font-size: 15px; text-decoration: none'>Openhab</a></h6></footer></body></html>"
+#define WEBPAGE_TEMPLATE_AP_HEADER      "<body><header class='apheader'><h1>Willkommen im kretz-net.</h1>"
```

Comment:
Do not change the default template in Custom-sample.h

Reply:
Sorry guys, that was a mistake from my side.
This is my Custom.h file. I will revert to the right Custom-sample.h